front/src/modules/ui/board/components/EntityBoardColumn.tsx

```diff
@@ -78,10 +86,12 @@ export const EntityBoardColumn = ({
         <BoardColumn
           onTitleEdit={handleTitleEdit}
           onDelete={onDelete}
+          column={column}
           title={column.title}
           color={column.colorCode}
           totalAmount={boardColumnTotal}
-          isFirstColumn={column.index === 0}
+          isFirstColumn={isFirstColumn}
+          isLastColumn={isLastColumn}
           numChildren={cardIds.length}
           stageId={column.id}
         >
```

Comment:
Prop drilling is getting a bit out of hand here, should we pass some of these into a context?

Reply:
Yes! you can create a BoardColumnContext and store everything there

Reply:
Done!